docs/product/prd.md

```
@@ -0,0 +1,106 @@
+# Product Requirements Document (PRD) – KinderTrack
+
+---
+
+## Índice
+
+1. [Breve descripción del producto](#breve-descripción-del-producto)  
+2. [Valor agregado y ventajas competitivas](#valor-agregado-y-ventajas-competitivas)  
+3. [Descripción general de las funcionalidades](#descripción-general-de-las-funcionalidades)  
+   1. [Registro de Asistencias](#registro-de-asistencias)  
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)